.github/workflows/release.yaml

```diff
@@ -0,0 +1,61 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+      - ci/publish-pipeline

```

Comment:
Can we remove this branch and leave just the `main` one?

Reply:
done